### ISSUE TITLE
[Accessibility] [DropDown] Invalid ARIA attribute values: aria-activedescendant

### ISSUE THREAD

**Issue by prmittal** [OfficeDev/office-ui-fabric-react#1043]
The drop down component fails accessibility validation for aria-valid-attr-value.

Details:
https://dequeuniversity.com/rules/axe/2.1/aria-valid-attr-value

Error displayed:
![image](https://cloud.githubusercontent.com/assets/11937034/23124463/33c5291a-f793-11e6-9870-877e35c91cee.png)

**Comment by prmittal** [comment]
Failure also observed for aria-required-children.

Details:
https://dequeuniversity.com/rules/axe/2.1/aria-required-children

Error displayed:
![image](https://cloud.githubusercontent.com/assets/11937034/23124516/8dac523c-f793-11e6-88bd-183ffc77fd2b.png)

**Comment by mdahamiwal** [comment]
1. `aria-valid-attr-value` is a valid issue since we set aria-activedescendant event when listbox is not available. Should be easily solvable.

2. `aria-required-children` requires children elements with textbox and listbox roles. This is not the case with this dropdown since we don't have a editable textbox and listbox is not always visible. Just verified https://jqueryui.com/selectmenu/ and it has the same issue.
